src/components/profile/Header.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+export default function Header() {
+    return (
+        <section className="relative block" style={{ height: '500px' }}>
```

Comment:
Could we use something like 
```
className="relative block h-96"
```
instead of the inline style? Or, we could add a new class named `h-125` which would add `height: 31.25rem`:
```
```
className="relative block h-125"
```
And somewhere inside the tailwind config we would have:
```
"125": "31.25rem"
```
31.25rem = 500px (if base is 16).